File: VB/WindowsApplication1/XtraForm1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace WindowsApplication1
	Partial Public Class XtraForm1
		Inherits DevExpress.XtraEditors.XtraForm
		Public Sub New(ByVal dataSource As Object)
			InitializeComponent()
			textEdit1.DataBindings.Add("EditValue", dataSource, "Field")
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click

		End Sub
	End Class
End Namespace

File: CS/WindowsApplication1/XtraForm1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace WindowsApplication1 {
    public partial class XtraForm1 : DevExpress.XtraEditors.XtraForm {
        public XtraForm1(object dataSource) {
            InitializeComponent();
            textEdit1.DataBindings.Add("EditValue", dataSource, "Field");
        }

        private void simpleButton1_Click(object sender, EventArgs e) {

        }
    }
}